appservice/src/intTest/groovy/nu/yona/server/BasicBuddyTest.groovy

```diff
@@ -117,7 +116,8 @@ class BasicBuddyTest extends AbstractAppServiceIntegrationTest
 		buddyConnectRequestMessages[0]._links.self.href.startsWith(YonaServer.stripQueryString(bob.messagesUrl))
 		buddyConnectRequestMessages[0]._links."yona:accept".href.startsWith(buddyConnectRequestMessages[0]._links.self.href)
 
-		appService.getUser(CommonAssertions.&assertUserGetResponseDetailsWithoutPrivateData, buddyConnectRequestMessages[0]._embedded."yona:user"._links.self.href)
+		// TODO YD-503 The user isn't a buddy yet, so that entity cannot be fetched
```

Comment:
Yes, to draw your attention and hear your opinion

Reply:
OK. I think we have to choose between:

1. Exclude the link because the user has no permissions to open it
2. Include the link and give permissions to users which have a pending buddy connect request

I think option 1 is easier.

Reply:
Indeed, but not easy :)
Option 2 is impossible, so I've implemented option 1.